From 36a6d6cee358cf1cdfcaeae9cb171f470285b8d5 Mon Sep 17 00:00:00 2001
From: Duy Dang <duy.dang.yw@renesas.com>
Date: Thu, 6 Jul 2023 18:53:48 +0900
Subject: [PATCH 14/14] domd: base-files: Set longer time to wait for devices

If booting from SD Card, it takes long time for devices
in /dev/aosvg/ to be available. Setting longer timeout
to prevent mounting failure.

Signed-off-by: Duy Dang <duy.dang.yw@renesas.com>
---
 .../recipes-core/base-files/base-files_%.bbappend         | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/meta-aos-rcar-gen4-domd/recipes-core/base-files/base-files_%.bbappend b/meta-aos-rcar-gen4-domd/recipes-core/base-files/base-files_%.bbappend
index 8887e03..397de5d 100644
--- a/meta-aos-rcar-gen4-domd/recipes-core/base-files/base-files_%.bbappend
+++ b/meta-aos-rcar-gen4-domd/recipes-core/base-files/base-files_%.bbappend
@@ -7,14 +7,14 @@ do_install_append() {
     # add Aos partitions
     echo "# Aos partitions">> ${D}${sysconfdir}/fstab
     echo "/dev/aosvg/downloads /var/aos/downloads  ext4 \
-        defaults,nofail,noatime,x-systemd.device-timeout=5s,x-systemd.after=var.mount  0  0" >> ${D}${sysconfdir}/fstab
+        defaults,nofail,noatime,x-systemd.device-timeout=10s,x-systemd.after=var.mount  0  0" >> ${D}${sysconfdir}/fstab
     echo "/dev/aosvg/workdirs  /var/aos/workdirs   ext4 \
-        defaults,nofail,noatime,x-systemd.device-timeout=5s,x-systemd.after=var.mount  0  0" >> ${D}${sysconfdir}/fstab
+        defaults,nofail,noatime,x-systemd.device-timeout=10s,x-systemd.after=var.mount  0  0" >> ${D}${sysconfdir}/fstab
     echo "/dev/aosvg/storages  /var/aos/storages   ext4 \
-        defaults,nofail,noatime,usrjquota=aquota.user,jqfmt=vfsv0,x-systemd.device-timeout=5s,x-systemd.after=var.mount \
+        defaults,nofail,noatime,usrjquota=aquota.user,jqfmt=vfsv0,x-systemd.device-timeout=10s,x-systemd.after=var.mount \
         0  0" >> ${D}${sysconfdir}/fstab 
     echo "/dev/aosvg/states    /var/aos/states     ext4 \
-        defaults,nofail,noatime,usrjquota=aquota.user,jqfmt=vfsv0,x-systemd.device-timeout=5s,x-systemd.after=var.mount \
+        defaults,nofail,noatime,usrjquota=aquota.user,jqfmt=vfsv0,x-systemd.device-timeout=10s,x-systemd.after=var.mount \
         0  0" >> ${D}${sysconfdir}/fstab
 
     # remove /run from fstab, run is mounted in initramfs
-- 
2.25.1

